boltdesignsystem/bolt
# DS 889: Drupal Admin UI Paternlab Section

## Jira

https://pegadigitalit.atlassian.net/browse/DS-889

## Summary

There's a new section Drupal Admin UI in the pattern lab navigation which contains the UX builds of the Drupal Admin panels for the new Layout Builder project.

## Details

**1. In the Bolt, we have a new section with UX builds:**
 - text paragraph
 - advanced text paragraph
 - icon paragraph
 - image paragraph
 - button paragraph
 - blockquote paragraph
 - list paragraph
 - action blocks paragraph
 - tabs and accordion paragraph
 
 These above were recreated from the private Codepen.
 
 **2. The animation tab was updated** 
 - the animation `type` dropdown was removed
 - the animation `style` was updated to have only the `fade` and `bounce` for now
 
:point_right: _**it's possible that this will be updated again in the future**_

## How to test

Pull the branch. Look at the Drupal admin UI section. Go thru all component examples and confirm that there are the same as Codepen examples.

## Release notes

We have a new Drupal admin UI section in the pattern lab navigation.
